src/styles/block/star.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
It can be done shorter.[Background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)